### PR TITLE
Update HLR & NOD dev docs

### PR DESCRIPTION
**Pull request by cilestin**
## Description of change
* More closely aligns HLR and NOD documentation for similar endpoints. 
* Places endpoints into the same order for consistency.

## Original issue(s)
https://vajira.max.gov/browse/API-2790
https://vajira.max.gov/browse/API-3989
https://vajira.max.gov/browse/API-4392

### CODE REVIEW

File: modules/appeals_api/app/swagger/appeals_api/v1/notice_of_disagreements_controller_swagger.rb [hunk 1]
```diff
@@ -25,52 +25,15 @@ class AppealsApi::V1::NoticeOfDisagreementsControllerSwagger
...
+  swagger_path '/notice_of_disagreements' do
     next unless PATH_ENABLED_FOR_ENV
 
     operation :post, tags: NOD_TAG do
       key :summary, 'Creates a new Notice of Disagreement.'
 
       description = 'Submits an appeal of type Notice of Disagreement.' \
-      'This endpoint is analogous to submitting VA Form 10182' \
-      'via mail or fax directly to the Board of Veterans’ Appeals.'
+      'This endpoint is the same as submitting [VA Form 10182](https://www.va.gov/vaforms/va/pdf/VA10182.pdf)' \
```
Comment (yakschuss) [inline modules/appeals_api/app/swagger/appeals_api/v1/notice_of_disagreements_controller_swagger.rb:35]: Does swagger support markdown this way? Huh. Had no idea!

**Review by yakschuss — APPROVED** [review]
LGTM 👍

File: modules/appeals_api/app/swagger/appeals_api/v1/notice_of_disagreements_controller_swagger.rb (hunk 1 above)
Comment (cilestin) [reply modules/appeals_api/app/swagger/appeals_api/v1/notice_of_disagreements_controller_swagger.rb:35]: It does indeed :)

**Review by nathanbwright — APPROVED** [review]
🚀

**Pull request merged**